src/main/java/com/synopsys/integration/detect/lifecycle/run/step/RapidModeStepRunner.java

```diff
@@ -97,20 +97,22 @@ private List<HttpUrl> parseScanUrls(String scanMode, SignatureScanOuputResult si
                 Reader reader = Files.newBufferedReader(Paths.get(scanOutputLocation));
 
                 SignatureScanResult result = gson.fromJson(reader, SignatureScanResult.class);
-                
-                Set<String> parsedIds = result.parseScanIds();
-                
-                for (String id : parsedIds) {
-                    HttpUrl url = new HttpUrl(blackDuckUrl + "/api/developer-scans/" + id);
-
-                    logger.info(scanMode + " mode signature scan URL: {}", url);
-                    parsedUrls.add(url);
+
+                if (result.getExitStatus() == null || !result.getExitStatus().equalsIgnoreCase("FAILURE")) {
+
+                    Set<String> parsedIds = result.parseScanIds();
+
+                    for (String id : parsedIds) {
+                        HttpUrl url = new HttpUrl(blackDuckUrl + "/api/developer-scans/" + id);
+
+                        logger.info(scanMode + " mode signature scan URL: {}", url);
+                        parsedUrls.add(url);
+                    }
                 }
```

Comment:
We should add a DEBUG log here:
`logger.debug(scanMode + " mode signature scan result not processed for scan IDs due to exist status from BD: " + result.getExitStatus());`